Update and complement kvb-hafas-mgate.json

1Maxnet1 opened the pull request:
When inspecting the request of the webapp, the options were present as I updated them and I added the missing option 'v'

1Maxnet1 committed:
Update and complement kvb-hafas-mgate.json

When inspecting the request of the webapp, the options were present as I updated them and I added the missing option 'v'

derhuerst commented:
> I added the missing option `v`

AFAIK most fields in `client`, including `client.v` are optional, but I don't mind including it.

1Maxnet1 commented:
> > I added the missing option `v`
> 
> AFAIK most fields in `client`, including `client.v` are optional, but I don't mind including it.

Yeah thought so. I added it anyway, as it is also available in other configs: https://github.com/public-transport/transport-apis/blob/31ebb2b28a6739f9c87cf681d34bbb45c3881154/data/de/vsn-hafas-mgate.json#L31
